code-challenges.test.js

```diff
@@ -10,50 +10,122 @@
+  // Expected output: "15 is divisible by three"
+  var num2 = 0;
+  // Expected output: "0 is divisible by three"
+  var num3 = -7;
+  // Expected output: "-7 is not divisible by three"
+  it("will use modulus to evaluate if the number has a remainder or not", () => {
+    // adding variables into the parameter since they are declared up top
+    expect(divisibleByThree(num1)).toEqual("15 is divisible by three");
+    expect(divisibleByThree(num2)).toEqual("0 is divisible by three");
+    expect(divisibleByThree(num3)).toEqual("-7 is not divisible by three");
+  });
+});
 
 // b) Create the function that makes the test pass.
 
+// create a function that takes in a number as the argument √
+// use modulus to see if it has a remainder √
+// return a string literal √
 
+const divisibleByThree = (num) => {
+  return num % 3 !== 0
+    ? `${num} is not divisible by three`
+    : `${num} is divisible by three`;
+}; // I have Prettier install and it's making it look like this, not sure if actually prettier...
 
 // --------------------2) Create a function that takes in an array of words and returns an array with all the words capitalized.
 
 // a) Create a test with expect statements for each of the variables provided.
-
-var randomNouns1 = ["streetlamp", "potato", "teeth", "conclusion", "nephew"]
-// Expected output: ["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"]
-var randomNouns2 = ["temperature", "database", "chopsticks", "mango", "deduction"]
-// Expected output: ["Temperature", "Database", "Chopsticks", "Mango", "Deduction"]
-
-
+describe("return array with all words capitalized", () => {
+  var randomNouns1 = ["streetlamp", "potato", "teeth", "conclusion", "nephew"];
+  // Expected output: ["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"]
+  var randomNouns2 = [
+    "temperature",
+    "database",
+    "chopsticks",
+    "mango",
+    "deduction",
+  ];
+  // Expected output: ["Temperature", "Database", "Chopsticks", "Mango", "Deduction"]
+  it("will use .map() and logic on all values in the array", () => {
+    // adding variables into the parameter since they are declared up top
+    expect(capitalize(randomNouns1)).toEqual([
+      "Streetlamp",
+      "Potato",
+      "Teeth",
+      "Conclusion",
+      "Nephew",
+    ]);
+    expect(capitalize(randomNouns2)).toEqual([
+      "Temperature",
+      "Database",
+      "Chopsticks",
+      "Mango",
+      "Deduction",
+    ]);
+  });
+});
 
 // b) Create the function that makes the test pass.
 
+// create a function that takes an array as argument √
+// use .map() to iterate over array and √
+// use charAt(0), .toUpperCase(), and .slice() in the rest of the word to make each value capitalized √
+// return array with a values capitalized √
+// at first I thought I only needed .toUpperCase() but that would have turned the entire word into all caps
 
+const capitalize = (array) => {
+  return array.map((value) => value.charAt(0).toUpperCase() + value.slice(1));
+};
```

Comment:
Looks great! It is perfect as is, but just as a fun thought you could shorten it even more by using `value[0]`